lineage/tHMM.py

```diff
@@ -1,15 +1,22 @@
 """ This file holds the parameters of our tHMM in the tHMM class. """
 
 import numpy as np
-import scipy.stats as sp
-from .Lineage_utils import get_numLineages, init_Population
+from .StateDistribution import StateDistribution, tHMM_E_init
 from .tHMM_utils import max_gen, get_gen
 
+class estimate:
```

Comment:
so u have to call the estimate class (which holds the thmm class) for every cell in the lineage right?

Reply:
No we instantiate the class once when the tHMM instantiation is called.

Reply:
the tHMM class holds one estimate class. The estimate class gets changed as thee various tHMM related functions are changed. It's similar to `paramlist` from before.